grafana-plugin/src/containers/AddResponders/parts/AddRespondersPopup/AddRespondersPopup.tsx

```diff
@@ -79,7 +79,7 @@ const AddRespondersPopup = observer(
         }
         setVisible(false);
       },
-      [isCreateMode, userStore, directPagingStore, setShowUserConfirmationModal, setVisible]
```

Comment:
- `userStore` is not consumed within the method passed to `useCallback` here
- the method accessed on `directPagingStore` will not change and thereby doesn't need to be passed as a dep here